### PR TITLE
Fixes: #80

### LINKED ISSUE #80: Create Incident Should not require incident_fields parameter
Create Incident Should not require incident_fields parameter

### PR DESCRIPTION
**Pull request by punkrokk**
Fixes #80, Fixes: #82, Fixes: #77

### CODE REVIEW

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/__version__.py  100
+ irflow_client/irflow_client.py  21
+ irflow_client/__init__.py  50
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/__init__.py  50
+ irflow_client/irflow_client.py  21
+ irflow_client/__version__.py  100
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/__init__.py  50
+ irflow_client/__version__.py  100
+ irflow_client/irflow_client.py  21
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/irflow_client.py  21
+ irflow_client/__version__.py  100
+ irflow_client/__init__.py  50
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/__version__.py  100
+ irflow_client/irflow_client.py  21
+ irflow_client/__init__.py  50
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/__version__.py  100
+ irflow_client/irflow_client.py  21
+ irflow_client/__init__.py  50
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/irflow_client.py  21
+ irflow_client/__version__.py  100
+ irflow_client/__init__.py  50
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/__init__.py  50
+ irflow_client/__version__.py  100
+ irflow_client/irflow_client.py  21
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/__version__.py  100
+ irflow_client/__init__.py  50
+ irflow_client/irflow_client.py  21
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/__version__.py  100
+ irflow_client/__init__.py  50
+ irflow_client/irflow_client.py  21
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/irflow_client.py  21
+ irflow_client/__version__.py  100
+ irflow_client/__init__.py  50
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/__init__.py  50
+ irflow_client/irflow_client.py  21
+ irflow_client/__version__.py  100
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/irflow_client.py  21
+ irflow_client/__version__.py  100
+ irflow_client/__init__.py  50
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/irflow_client.py  21
+ irflow_client/__init__.py  50
+ irflow_client/__version__.py  100
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/irflow_client.py  21
+ irflow_client/__init__.py  50
+ irflow_client/__version__.py  100
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

File: examples/01_run_all_api_calls.py [hunk 1]
```diff
@@ -243,12 +253,15 @@
 description = 'Super Bad API Incident - Remediated'  # The Alert Description
 
 owner_id = 2
-group_ids = [1,2,3]
+group_ids = [1, 2,  3]
```
Comment (rmarkle) [inline examples/01_run_all_api_calls.py]:
```suggestion
group_ids = [1, 2, 3]
```

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/irflow_client.py  21
+ irflow_client/__init__.py  50
+ irflow_client/__version__.py  100
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/irflow_client.py  21
+ irflow_client/__init__.py  50
+ irflow_client/__version__.py  100
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/__version__.py  100
+ irflow_client/__init__.py  50
+ irflow_client/irflow_client.py  21
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Comment by punkrokk** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Coverage increased per file
===========================
+ irflow_client/irflow_client.py  21
+ irflow_client/__init__.py  50
+ irflow_client/__version__.py  100
         

Clones removed
==============
+ examples/01_run_all_api_calls.py  -2
         
```


See the complete overview on [Codacy](https://app.codacy.com/app/Syncurity/irflow-sdk-python/pullRequest?prid=3886089&bid=13532504)

**Pull request merged**